src/functions-patterns.php

```diff
@@ -48,6 +48,19 @@ function register_patterns() {
 		return;
 	}
 
+	// Query all pattern categories.
+	$pattern_categories = new WP_Term_Query( [
+		'taxonomy' => 'bpb_pattern_category',
+		'fields'   => 'all',
+ 	] );
+
+	foreach ( $pattern_categories->terms as $term ) {
```

Comment:
There may not be any terms yet. Probably needs another check right?

Reply:
Fixed